fetch-api/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@shoutem/js-api",
```

Comment:
No problem. Just a question regarding `react-`. There is nothing react specific, these are dependencies
```
 "@shoutem/js-api": "file:../fetch-api",
    "@shoutem/js-api": "0.1.1-beta.2",
    "@shoutem/redux-composers": "0.1.3",
    "@shoutem/redux-io": "2.0.2-beta.1",
    "flux-standard-action": "^1.2.0",
    "lodash": "^4.17.4",
    "urijs": "^1.18.9"
```